commonblocks/static/commonblocks/js/hallo-bootstrap.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * registerHalloPlugin() applies to all editor instances of a Page.
+ * We want to encapsulate custom plugin configuration, per field.
+ */
+function makeHalloSimpleRichTextEditable(id, plugins) {
+    var defaultPlugins = $.extend({}, halloPlugins);
+
+    if (typeof plugins === 'object') {
```

Comment:
Checking `typeof === 'object'` is quite dangerous, because in JS `typeof null === 'object'`. If you _really_ want to check this is an object, you're in luck because there is `$.isPlainObject`.